libs/langchain/langchain/vectorstores/redis/base.py

```diff
@@ -952,19 +1054,16 @@ def _collect_metadata(self, result: "Document") -> Dict[str, Any]:
 
     def _prepare_query(
         self,
-        query: str,
+        query_embedding: List[float],
```

Comment:
why the parameter name change here?

Reply:
I think it would be a more appropriate name given the change from a string query to just its embedding (a list).